tools/c7n_azure/c7n_azure/resources/key_vault_keys.py

```diff
@@ -98,6 +98,16 @@ class resource_type(ChildTypeInfo):
         parent_manager_name = 'keyvault'
         raise_on_exception = False
 
+        id = 'kid'
+        name = 'kid'
+
+        default_report_fields = (
+            'kid',
+            'attributes.enabled',
+            'attributes.exp',

```

Comment:
Does report display it as a date?

Reply:
When I was looking at the cassettes I did not even see this field? 

```
                                "kid": "https://cckeyvault1c6ra5qqpkhceo.vault.azure.net/keys/cctestec",
                                "attributes": {
                                    "enabled": true,
                                    "created": 1557245473,
                                    "updated": 1557245473,
                                    "recoveryLevel": "Purgeable"
                                }
```

If it is only there in some cases it should be OK I guess, as JMESPath will just return a `None`.

Reply:
It returns it as a unix timestamp. I had the JMESPath set up to return "Non-expiring" if there was no expiration date, but then the column header looks like a mess and isn't clear that it is the expiration date anymore

Reply:
I think that is fine, JMESPath doesn't know how to format dates, we'd need the output class to do this or we'd need to augment it somehow.  

It may not make sense in the grid output, but for JSON output that is parsed elsewhere it still makes sense.